modules/exploits/windows/local/cve_2020_0787_bits_arbitrary_file_move.rb

```diff
@@ -0,0 +1,242 @@
+##
+# This module requires Metasploit: https://metasploit.com/download
+# Current source: https://github.com/rapid7/metasploit-framework
+##
+
+require 'msf/core/post/common'
+require 'msf/core/post/windows/priv'
+require 'msf/core/post/windows/registry'
+require 'msf/core/exploit/exe'
+require 'msf/core/post/windows/filesystem'
+require 'msf/core/exploit/file_dropper'
+require 'msf/core/post/file'
+
+class MetasploitModule < Msf::Exploit::Local
+  Rank = ExcellentRanking
```

Comment:
Double-check this?

Reply:
This is a DLL hijacking issue and should never crash the service, both BITS and USO, so I don't see why this wouldn't have this rating? Unless your thinking of some other reason to lower it in which case I'm open to opinions :)

Reply:
Cool. I thought you said you had some issues developing this module, but if you've cleared that all up, this is fine.